optuna/storages/_rdb/models.py

```diff
@@ -80,11 +73,6 @@ def find_or_raise_by_name(cls, study_name: str, session: orm.Session) -> "StudyM
 
         return study
 
-    @classmethod
-    def all(cls, session: orm.Session) -> List["StudyModel"]:
-
-        return session.query(cls).all()
-
 
 class StudyDirectionModel(BaseModel):
```

Comment:
How about deleting `StudyDirectionModel.where_study` as well. It's seemingly unused.